src/commands/azureServiceOperators/helpers/azureservicehelper.ts

```diff
@@ -0,0 +1,287 @@
+    const styleUri = stylePathOnDisk.with({ scheme: 'vscode-resource' });
+    const pathUri = htmlPathOnDisk.with({ scheme: 'vscode-resource' });
+
+    const htmldata = fs.readFileSync(pathUri.fsPath, 'utf8').toString();
+    const certManagerResult = installationResponse.installCertManagerResult;
+    const certManagerSatatusResult = installationResponse.checkCertManagerRolloutStatusResult;
+    const issuerCertResult = installationResponse.installIssuerCertResult;
+    const olmCrdResult = installationResponse.installOlmCrdResult;
+    const olmResult = installationResponse.installOlmResult;
+    const operatorResult = installationResponse.installOperatorSettingsResult;
+    const operatorSettingsResult = installationResponse.installOperatorResult;
+    const getOperatorsPodResult = installationResponse.getOperatorsPodResult;
+
+    let exitCode = 0;
+
+    // If total of outputCode is more than 0 then obviously something failed and display the html error, with std ouput.
+    exitCode = installationExitCodeResponse(exitCode, certManagerResult);
+    exitCode = installationExitCodeResponse(exitCode, certManagerSatatusResult);
+    exitCode = installationExitCodeResponse(exitCode, issuerCertResult);
+    exitCode = installationExitCodeResponse(exitCode, olmCrdResult);
+    exitCode = installationExitCodeResponse(exitCode, olmResult);
+    exitCode = installationExitCodeResponse(exitCode, operatorResult);
+    exitCode = installationExitCodeResponse(exitCode, operatorSettingsResult);
+    exitCode = installationExitCodeResponse(exitCode, getOperatorsPodResult);
+
+    htmlHandlerRegisterHelper();
+    const template = htmlhandlers.compile(htmldata);
+    const data = {
+        cssuri: styleUri,
+        name: clustername,
+        certManagerOutput: installationStdResponse(certManagerResult),
+        certManagerRolloutStatus: installationStdResponse(certManagerSatatusResult),
+        issuerOutput: installationStdResponse(issuerCertResult),
+        installOlmCrdOutput: installationStdResponse(olmCrdResult),
+        installOlmOutput: installationStdResponse(olmResult),
+        installOperatorOutput: installationStdResponse(operatorResult),
+        asoSettingsOutput: installationStdResponse(operatorSettingsResult),
+        getOperatorsPodOutput: installationStdResponse(getOperatorsPodResult),
+        outputCode: exitCode
+    };
+    const webviewcontent = template(data);
+
+    return webviewcontent;
+}
+
+function installationStdResponse(
+    installShellResult: k8s.KubectlV1.ShellResult | undefined
+): string | undefined {
+    // Standard output code of all the installation response.
+    return installShellResult ? installShellResult.stderr + installShellResult.stdout : undefined
+}
+
+function installationExitCodeResponse(
+    exitCode: number,
+    installShellResult: k8s.KubectlV1.ShellResult | undefined
+): number {
+
+    if (exitCode === 0 && installShellResult) {
```

Comment:
This means that if `installShellResult` is undefined (which means we couldn't even run the program) you will leave the exit code at 0; that's probably not what you want.

Reply:
I tactfully note that the `all(...)` version of the process would not have this bug. *grin*